File: data/scripts/hook/debugstall.lua
local time = 10;

if (appl_arguments) then

	for i,v in ipairs(appl_arguments()) do
		if string.sub(v, 1, 11) == "debugstall=" then
			local rem = string.sub(v, 12)
			local num = tonumber(rem)
			if num and num > 0 then
				left = num
			end
		end
	end

end

frameserver_debugstall(time);
warning("debugstall set to " .. tostring(time));

File: data/scripts/hook/debugstall.zig

var time = 10;

pub fn __init() void {
    if (appl_arguments) {
        for (appl_arguments(), 0..) |v, i| {
            if (string.sub(v, 1, 11) == "debugstall=") {
                var rem = string.sub(v, 12);
                var num = tonumber(rem);
                if (num and (num > 0)) {
                    left = num;
                }
            }
        }
    }
    frameserver_debugstall(time);
    warning("debugstall set to " ++ tostring(time));
}
